src/transformers/modeling_utils.py

```diff
@@ -1713,6 +1720,10 @@ def save_pretrained(
         """
         use_auth_token = kwargs.pop("use_auth_token", None)
 
+        # For backward compatibility with PEFT library all keys of the state dict of adapters needs to be pre-pended with
+        # `base_model.model`. Users can disable this behaviours by setting `save_peft_format` to `False`.
```

Comment:
Should this parameter not be documented in the docstring?

Reply:
I added `save_peft_format` in the signature of `save_pretrained` method. I am not sure if this is the correct approach, I think usually we don't document what is inside kwargs . Maybe @sgugger can comment on this, if this is not what we usually do I am happy to revert [a8eb928](https://github.com/huggingface/transformers/pull/25077/commits/a8eb92838d98e465849327d4466f2fefc17549f6) and add `save_peft_model` in the docstring.

Reply:
Ah I see, then it might be enough as is.